Added tile for azure active directory (#7102)
DataDog/integrations-core: azure_active_directory/README.md

### Configuration

1. Set up the log forwarding pipeline from Azure to Datadog using Event Hubs by following the [log collection documentation][2].

2. In Azure portal, select _Azure Active Directory > Monitoring > Audit logs_.
   
3. Select **Export Settings**.

4. In the Diagnostics settings pane, do one of the following:

   - To change existing settings, select **Edit setting**.
   - To add new settings, select **Add diagnostics setting**. You can have up to three settings.

5. Select the **Stream to an event hub** check box, and then select **Event Hub/Configure**.

6. Select the Azure subscription and Event Hubs namespace that you created earlier to route the logs to.
   
7. Select OK to exit the event hub configuration.

8. Do one or both of the following. Datadog recommends selecting both.

   - To send audit logs, select the **AuditLogs** check box.
   - To send sign-in logs, select the **SignInLogs** check box.
  
9. Select **Save**.

Logs should start coming into Datadog within 15 minutes.
For more details on the setup, see the [Azure tutorial][3].

## Data Collected

### Logs

This integration allows you to setup log ingestion for Azure Active Directory activity logs.

> This inlcudes

Reviewer: ```suggestion
This includes the following:
```

typo

# Agent Check: azure_active_directory

## Overview

Azure Active Directory is a cloud hosted Active Directory offering by Microsoft Azure.
This integration allows you to ingest your [Azure AD activity logs][1] (Audit and SignIn logs) to Datadog.

## Setup

### Installation

This integration uses log forwarding from Azure using Event hub to Datadog and cofiguring Azure AD to forward activity logs to the event hub.

### Configuration

1. Setup the log forwarding pipeline from Azure to Datadog using Event Hub by following [directions here][2]

2. In Azure portal, Select Azure Active Directory > Monitoring > Audit logs.
   
3. Select Export Settings.

4. In the Diagnostics settings pane, do either of the following:

>    - To change existing settings, select Edit setting.

Reviewer 2: ```suggestion
   - To change existing settings, select **Edit setting**.
```

>    - To add new settings, select Add diagnostics setting. You can have up to three settings.

Reviewer 2: ```suggestion
   - To add new settings, select **Add diagnostics setting**. You can have up to three settings.
```

> 5. Select the Stream to an event hub check box, and then select Event Hub/Configure.

Reviewer 2: ```suggestion
5. Select the **Stream to an event hub** check box, and then select **Event Hub/Configure**.
```

6. Select the Azure subscription and Event Hubs namespace that you created earlier to route the logs to.
   
7. Select OK to exit the event hub configuration.

> 8. Do either or both of the following:

Reviewer 2: ```suggestion
8. Do one or both of the following. Datadog recommends selecting both.
```

>    - To send audit logs, select the AuditLogs check box.

Reviewer 2: ```suggestion
   - To send audit logs, select the **AuditLogs** check box.
```

>    - To send sign-in logs, select the SignInLogs check box.

Reviewer 2: ```suggestion
   - To send sign-in logs, select the **SignInLogs** check box.
```

>     we recommend that you select both

Reviewer 2: ```suggestion
```

> 9. Select Save to save the setting.

Reviewer 2: ```suggestion
9. Select **Save**.
```

> Logs should start coming into the platform after about 15 minutes.

Reviewer 2: ```suggestion
Logs should start coming into the Datadog within 15 minutes.
```

Author: Should probably say "Logs should start coming into Datadog within 15 minutes." (No "the")

Author: Changed.

> For more details on the setup find [directions here][3]

Reviewer 2: ```suggestion
For more details on the setup, see the [Azure tutorial][3].
```

## Data Collected

### Logs

This integration allows you to setup log ingestion for Azure Active Directory activity logs.

This inlcudes

>    - Sign-ins – Provides information about the usage of managed applications and user sign-in activities

Reviewer 2: ```suggestion
   - Sign-ins – Provides information about the usage of managed applications and user sign-in activities.
```
